content/authors/eric-bigelow/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Eric Bigelow
+nicknames: Eric J. Bigelow
+avatar_filename: img-1556.jpg
+bio: I'm interested in using experimentation, modeling, and theory to explore
+  the relationship between intuition and formal reasoning in cognition, and how
+  people learn concepts in abstract domains such as science, programming, and
+  mathematics.
+user_groups:
+  - Graduate Students
```

Comment:
You should also add "Researchers" as a role. It's a bit redundant, but has some uses. 😅